tsdb/store.go

```diff
@@ -155,6 +155,12 @@ func (s *Store) loadShards() error {
 				return err
 			}
 			for _, sh := range shards {
+				// Check to see if there are any `.tmp` directories that were left over from failed shard snapshots
+				if sh.IsDir() && strings.HasSuffix(sh.Name(), ".tmp") {
+					if err := os.Remove(sh.Name()); err != nil {
```

Comment:
I'm unsure of this.  The documentation doesn't state, and the code just calls the system call.  This means it could be dependent on the system architecture possible? Better safe than sorry?